### DISCUSSION TITLE
Remove "bite" from highly rated-packages that no longer work

### DISCUSSION THREAD

**Discussion by marcusmoore** [tighten/novapackages#135] [Ideas]
Issue #6 included the following line: `Packages still feel too much like a roll of the dice. Just bit by another 5 star rated package that just doesn't work.`

We should figure out a way to show users the "health" of a package. @darkboywonder and I think that a good solution will be to include the open issue count in the right hand sidebar as an indicator for users to say "this package has a 5 star rating but 12 open issues...I should see what those issues are".

There is the possibility to enhance this further by generating some kind of "health" score for the package by checking how long issues have remained open.

We're also open to other ideas to address this "bite". Any thoughts @mattstauffer?

**Comment by ralphschindler** [comment]
I agree, I often have to do much diligence in seeing if a package was just a fire-and-forget project, or if it is actually actively maintained. There should be a status for healthy packages.
